iothub/device/src/Message.cs

```diff
@@ -63,6 +63,22 @@ public Message(byte[] byteArray)
             _streamDisposalResponsibility = StreamDisposalResponsibility.Sdk;
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="messagePayload"></param>
+        /// <param name="telemetryConvention"></param>
+        public Message(object messagePayload, TelemetryConvention telemetryConvention = default)
+            : this(new MemoryStream(GetTelemetryConventionInstance(telemetryConvention).GetObjectBytes(messagePayload)))
+        {
+            TelemetryConvention convention = GetTelemetryConventionInstance(telemetryConvention);
```

Comment:
Do you intend to use GetTelemetryConventionInstance elsewhere in the code? Seems like we could make a private member and set to the constructor instance or the default one.

Reply:
GetTelemetryConventionInstance is being used only here, so any kind of refactoring is welcome!

Reply:
Gone